tests/test_e2e.py

```
@@ -1254,3 +1256,194 @@ def test_list_re_replace():
+        assert "Received 2 arguments to function 'floor', expected at most 1" in str(e)
+
+
+def test_log():
+    res = query("RETURN log(1) AS name")
+    assert res.result_set == [[0]]
+
+    res = query("RETURN log(0) AS name")
+    assert res.result_set == [[float('-inf')]]
+
+    res = query("RETURN log(-1) AS name")
+    v = res.result_set[0][0]
```

Ruff (0.8.2)

Local variable `v` is assigned to but never used

Remove assignment to unused variable `v`

(F841)